docs/source/architecture.rst

```diff
@@ -402,32 +402,83 @@ exceeds this value, Sawtooth PBFT uses these rules to prune the log:
-the timer and proceeds as normal. If the timer expires, it considers the primary
-node to be faulty and requests a view change by sending a ``ViewChange``
-message.  However, view changing mode does not occur until enough other nodes
-agree (send their own ``ViewChange`` messages).
-
-View changing mode has the following steps:
-
-1. Any node who decides the primary is faulty sends a ``ViewChange`` message to
-   all nodes. This message contains the node’s current sequence number (block
-   number) and  its current view.
-
-#. After sending the ``ViewChange`` message, the node enters View Changing mode.
-
-#. Once a node receives :math:`2f + 1` ``ViewChange`` messages (including
-   its own), it changes its own view to :math:`v + 1`, and resumes Normal
-   operation.
-
-The next primary node is determined by the node ID, in sequential order, based
-on the order of nodes in the ``sawtooth.consensus.pbft.members`` on-chain setting.
-For more information, see :ref:`view-changes-choosing-primary-label`.
+A `view change` switches to a different primary node. A node starts a new view
+change if any of the following occur:
+
+1. The `idle timeout` expires - when a node enters the ``PrePreparing`` phase,
+   it will start its idle timeout. If the node receives a new block and a
+   matching ``PrePrepare`` from the primary for its current sequence number
+   before the timeout expires, it will stop the timeout; if not, the node will
+   initiate a view change when the timeout expires.
+
+2. The `commit timeout` expires - when a node enters the ``Preparing`` phase, it
+   will start its commit timeout. If the node is able to move on to the
+   ``Finishing`` phase and send a request to the validator to commit the block
+   before the timeout expires, it will stop the timeout; if not, the node will
+   initiate a view change when the timeout expires.
+
+3. The `view change timeout` expires - when a node starts a view change to view
+   ``v``, it will start a view change timeout. If the node is able to complete
+   the view change before the timeout expires, it will stop the timeout; if not,
+   it will initiate a new view change to view ``v + 1``.
+
+4. Multiple ``PrePrepare`` messages are received for the same view and sequence
+   number but different blocks - this indicates the primary is faulty, since
+   this behavior is invalid.
+
+5. A ``Prepare`` message is received from the primary - this indicates the
+   primary is faulty, since this behavior is invalid.
+
+6. ``f + 1`` ``ViewChange`` messages are received for the same view - this
+   ensures that a node does not wait too long to start a view change; since only
+   ``f`` nodes (at most) can be faulty at any given time, if more than ``f``
+   nodes decide to start a view change, other nodes can safely join them to
+   perform that view change.
+
+To start a view change, a node will do the following:
+
+- Update its mode to ``ViewChanging(v)``, where ``v`` is the view the node is
```

Comment:
Use numbers instead of bullets, because the node does these things in the given order (at least, that's my assumption).